src/eval-frame/components/reps/__tests__/value-renderer.test.js

```diff
@@ -9,6 +9,22 @@ import ErrorRenderer from "../error-handler";
 import HTMLHandler from "../html-handler";
 import UserReps from "../user-reps-manager";
 
+import { IODIDE_EVALUATION_RESULTS } from "../../../global-state-extras";
+
+const simpleTestCases = [
+  { type: "array", val: [1, 2, 3.4] },
+  { type: "object", val: { a: 1, b: 2 } },
+  { type: "number", val: 12345 },
+  { type: "string", val: "foo" },
+  { type: "null", val: null },
+  { type: "undefined", val: undefined },
+  { type: "function", val: () => 123 }
+];
+
+simpleTestCases.forEach((testCase, i) => {
+  IODIDE_EVALUATION_RESULTS[`result_${i}`] = testCase.val;
+});
+
```

Comment:
updating tests now that ValueRenderer takes a valueKey prop that it uses to look up values in IODIDE_EVALUATION_RESULTS